dev/sg/internal/db/db.go

```diff
@@ -47,7 +47,7 @@ var (
 		Name:            "frontend",
 		MigrationsTable: "schema_migrations",
 		FS:              GetFSForPath("frontend"),
-		DataTables:      []string{"lsif_configuration_policies"},
+		DataTables:      []string{"lsif_configuration_policies", "roles"},
```

Comment:
I think this is easier for future readers:

```suggestion
		DataTables:      []string{"lsif_configuration_policies", "rbac_roles"},
```

Reply:
The table name is `roles`, is it necessary to prefix this with `rbac` considering this is the first of it's kind and this system handles all things relating to roles and permissions within sourcegraph.

Reply:
Agreed. No change to this.